### ISSUE TITLE
At least one application did not load

### ISSUE THREAD

**Issue by reidka** [uoft-tapp/old-orig-tapp#185]
I learned by accident of an application from a student that did not get loaded.  I think this is a problem with the importer?  The only thing I can see is that the "ta_experience field contains non-ascii characters.

I'll share the data with @michellemtchai so you can try to figure out why it failed.  I suspect this is another case where we need clear error messages instead of silent failure.

**Comment by michellemtchai** [comment]
@reidka Where is this error showing? Is it in the console of the browser? Or is it on of those red errors from Rails? I would like to see a screenshot of the error if possible. You can post it on Slack and redact it if there's sensitive information involved. Thanks.

**Comment by reidka** [comment]
There was no error message.  I found out because it was a case where the applicants were also supposed to contact the instructor, so the instructor wanted to hire a person that I couldn't see in tapp.  I looked at the JSON file I imported from CHASS and his application was there, so it didn't get imported.

I'm looking for other ones.  Okay, this looks like a big problem.  The app says there are 486 applicants, which squares with the number of "app_id" labels in the JSON, but I only see 403 applicants in the app  (by cutting and pasting the unassigned list and the assigned list into a text editor and counting lines -- so it might be a bit error prone.)

Since the number of applicant in the summary view is correct, it might not be the importer.  I
